lib/frontman/context.rb

```diff
@@ -121,5 +118,27 @@ def load_buffer
         instance_variable_get(:@_erbout)
       end
     end
+
+    def set_content_buffer(content)
```

Comment:
I'm not sure about the naming of this method. What are your thoughts on renaming it to this?

```suggestion
    def load_from_buffer(content)
```

Reply:
I think thats a bit too similar to the method above it called `loader_buffer` it should be a bit more descriptive